__init__.py

```diff
@@ -580,11 +580,24 @@ def _handle_sensor(self, message):
                 "value": sensor_state,
                 "current_temp": current_temp,
                 "targeted_temp": target_temp})
+
+            self._display_sensor_dialog(
+                sensor_name,
+                attributes['current_temperature'],
+                sensor_state
+            )
         elif domain == "cover":
             self.speak_dialog(f'homeassistant.sensor.cover.{sensor_state}', data={
                 "dev_name": sensor_name})
+
+            sensor_states = self.translate_namedvalues(
+                'homeassistant.sensor.cover.state')
```

Comment:
Could move these to `initialize()` so we only have to do it once

Reply:
Tests are passing so I'll merge this - and refactoring can happen in a follow up PR if there's interest in it.